set.mm

```diff
@@ -528217,7 +528221,34 @@ sethood hypotheses (compare ~ opth ).  (Contributed by BJ, 6-Oct-2018.) $)
 
 $(
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
-                Elementwise intersection (families of sets induced on a subset)
+  Elementwise operations
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $( Token for elementwise operations. $)
+  $c elmtwise $.
+
+  $( Syntax for elementwise operations. $)
+  celmtwise $a class elmtwise $.
+
+  ${
+    $d o x y z u v $.
+    $( Define the elementwise operation associated with a given operation.  For
+       instance, ` + ` is the addition of complex numbers ( ~ axaddf ), so if
+       ` A ` and ` B ` are sets of complex numbers, then
+       ` ( A ( elmtwise `` + ) B ) ` is the set of numbers of the form
+       ` ( x + y ) ` with ` x e. A ` and ` y e. B ` .  The set of odd natural
+       numbers is
+       ` ( ( { 2 } ( elmtwise `` x. ) NN0 ) ( elmtwise `` + ) { 1 } ) ` , or
+       less formally ` 2 NN0 + 1 ` .  (Contributed by BJ, 22-Dec-2021.) $)
+    df-elmtwise $a |- elmtwise = ( o e. _V |->
+       ( x e. _V , y e. _V |-> { z | E. u e. x E. v e. y z = ( u o v ) } ) ) $.
+  $}
+
```

Comment:
This looks interesting, but I see no benefit regarding odd numbers being defined directly as { z e. ZZ | ( ( z + 1 ) / 2 ) e. ZZ } or { z e. ZZ | -. 2 || z }.

Reply:
It makes the definitions more uniform, and is close to the standard symbol 2 NN + 1 (see e.g. Conjecture 1.5 of https://arxiv.org/pdf/1909.03562.pdf).